### ISSUE TITLE
Traveling Salesman Problem

### ISSUE THREAD

**Issue by matthewrobb** [shalzuth/SanctuaryMaps#6]
Can we derive enough information to generate an optimal step by step path to getting all the altars?

see: https://jeanropke.github.io/RDR2CollectorsMap/

Seems like with the map points being coordinate based we would just need to know which coordinates are impassable. Also it would be helpful to know where jump-points are, specially those that a horse cannot make.

**Comment by shalzuth** [comment]
Is following the roads good enough for pathfinding? This would be the easiest to dump/parse/follow?
If someone can commit to add code to pathfind to the altars, I can add in that data.

**Comment by matthewrobb** [comment]
Yeah I think following the roads would be good if it at least gets you in the neighborhood of the altar. Going beyond altars it would be nice to build semi-intelligent pathfinding for anything being tracked. The pin/marker in-game doesn't even consider waypoints when directing you. Would love to be in a position to set my starting point and then get that ideal route to any number of other points taking into consideration roads, waypoints, and optionally jump points like skipping the climb spots.

**Comment by matthewrobb** [comment]
Saw this today: https://www.reddit.com/r/diablo4/comments/14dtis4/altar_of_lilith_peregrination_get_all_the_altars/

![map](https://i.redd.it/mcjgk482127b1.jpg)
